### PR TITLE
Review youtube-embed project

### PR DESCRIPTION
**Pull request by Lawmsangi**
This is a simple youtube player page build with html and css

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+.header #toggler{
+    display: none;
+}
+
+.header__left{
+    display: flex;
+     flex: 0.1;
+}
+
+.header__left fa-bars{
```
Comment (Lawmsangi) [inline index.css:44]: It seems here that you are not using proper selector so the style will not be applied, try using .header__left  .fas  .fa-bars .

File: index.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    width: 30%;
+    margin-left: 10px;
+}
+
+.content-image{
+    position: relative;
+    width: 28rem;
+    cursor: pointer;
+    padding: 10px;
+}
```
Comment (Lawmsangi) [inline index.css:137]: It would be better to add a single space between your styles to make code more clean.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    <!-- main-section -->
+    <main>
+        <section class="main-video">
+            <iframe width="747" height="455" src="https://www.youtube.com/embed/w7ejDZ8SWv8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </section>
+
+        <section class="sidebar">
+            <div class="content-video">
+                <div class="content-image">
+                    <img src="dotandkey.png" alt="">
```
Comment (Lawmsangi) [inline index.html:49]: Please add alt text to images to follow best html practises.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                    <h3>Extra 20% Off Use Code: NEW20</h3>
+                    <h4>Helps with better vitamin D absorption, making it beneficial to be in the sun</h4>
+                    <h4>Ad <span> :Dot and Key</span></h4>
+                </div>
+            
+                <div class="content-image">
+                    <img src="playlist.jpg" alt="">
```
Comment (Lawmsangi) [inline index.html:56]: Please add alt text to images to follow best html practises.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+                    <h3>070 Shake - Guilty Conscience (Official Video)</h3>
+                    <h4>070 Shake, Jaden Smith, Bakar and more</h4>
+                    <h4>10M views <span> :3 years ago</span></h4>
+                </div>
+            
+                <div class="content-image">
+                    <img src="lofi.webp" alt="">
```
Comment (Lawmsangi) [inline index.html:63]: Please add alt text to images to follow best html practises.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                    <h3>lofi hip hop radio - beats to relax/study to</h3>
+                    <h4>Lofi Girl <i class="fa-sharp fa-solid fa-circle-check"></i></h4>
+                    <h4>53K watching</h4>
+                    <i class="fa-solid fa-signal-stream"></i>
+                </div>
+            
+                <div class="content-image">
+                    <img src="paramore.jpg" alt="">
```
Comment (Lawmsangi) [inline index.html:71]: Please add alt text to images to follow best html practises.

**Review by Lawmsangi — COMMENTED** [review]
Very nice work on the code. Good job!
But there are some changes required to complete the project.

**Review by AbelG101 — APPROVED** [review]
# Hi @Lawmsangi 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Comments are phrased politely, as suggestions instead of orders ✔
- Comments are fairly clear and easy to understand. ✔
- Proper explanation for each requested change ✔

## Optional suggestions
- [OPTIONAL] It would be better if you can use emojis in your comments as it makes the code review a bit more human-friendly 👽🤖.
- [OPTIONAL] I'd also suggest you provide a more detailed explanation for each requested change. for example, the statement "...to follow best html practices" is a bit vague you could instead say "The reason why it's important to provide an `alt` text is that it helps people with visual impairments to understand the content of the image. Alternative text is read aloud by screen readers and other assistive technologies, allowing people who can't see the image to understand what it depicts."

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**